src/packages/core/src/servers/http-server.ts

```diff
@@ -93,10 +94,57 @@ function sendResponse(
   });
 }
 
-export type HttpServerOptions = Pick<InternalOptions["server"], "rpcEndpoint">;
+function sendChunkedResponse(
```

Comment:
I think the code comments are pretty decent here, but an explanation and walk-through over zoom may help in the review, because I still find it easy to get lost here. :-)